### PR TITLE
 [12.0][MIG] storage_thumbnail

### PR DESCRIPTION
**Pull request by rousseldenis**
Includes:

* [x] #69 

Added tests

### CODE REVIEW

**Comment by rousseldenis** [comment]
Coverage decreased because there was no tests before

File: storage_thumbnail/tests/test_thumbnail.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+from .models import ModelTest
+
+
+class TestStorageThumbnail(TransactionComponentCase):
+    def setUp(self):
+        super().setUp()
+
+        # Register model inheritance
+        ModelTest._build_model(self.env.registry, self.env.cr)
```
Comment (lmignon) [inline storage_thumbnail/tests/test_thumbnail.py:16]: @rousseldenis You don't cleanup the registry at the end of your tests..... https://github.com/OCA/search-engine/blob/12.0/connector_search_engine/tests/models_mixin.py#L51

**Review by lmignon — CHANGES_REQUESTED** [review]

File: storage_thumbnail/tests/test_thumbnail.py (hunk 1 above)
Comment (rousseldenis) [reply storage_thumbnail/tests/test_thumbnail.py:16]: Good catch. I forgot teardown

Comment (rousseldenis) [reply storage_thumbnail/tests/test_thumbnail.py:16]: @lmignon Done

**Review by lmignon — APPROVED** [review]

**Pull request merged**